assets/charts/components/cert-manager/templates/letsencrypt-clusterissuer-prod.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: cert-manager.io/v1alpha2
+apiVersion: cert-manager.io/v1
```

Comment:
I think this can be merged into original commit.

Reply:
I think it is okay to highlight because these changes don't come from upstream.

Reply:
They are atomic to the upgrade process, but OK.